e2e/test_operator/operator_test.go

```diff
@@ -142,6 +144,66 @@ var _ = Describe("Operator", Label("e2e"), func() {
 		})
 	})
 
+	When("taint a Pod", func() {
```

Comment:
```suggestion
	When("taint a node is tainted that hosts a Pod", func() {
```

We cannot taint Pods.